firmware/config/boards/microrusefi/connectors/generated_ts_name_by_pin.cpp

```diff
@@ -25,10 +25,10 @@ const char * getBoardSpecificPinName(brain_pin_e brainPin) {
 		case GPIOD_1: return "12 - Ignition 4";
 		case GPIOB_1: return "40 - AN volt 9, Aux Reuse";
 		case GPIOB_0: return "36 - AN volt 8, Aux Reuse";
-		case GPIOB_9: return "AUX J2 PB9";
 		case GPIOD_7: return "14 - GP Out 5";
-		case GPIOB_8: return "AUX J2 PB8";
+		case GPIOB_9: return "AUX J2 PB9";
```

Comment:
GRRRRRR

I am very concerned when people do not care what they include in PRs :(

Reply:
I don't think it was me.

Reply:
I haven't pushed this on.

[Auto-generated configs and docs](https://github.com/rusefi/rusefi/pull/3988/commits/0ce9438397c00070dcee53641612e8d447f94404)

Reply:
you did push it, because you mis-merged it from master

Reply:
You are totally right I am just being defensive and aggressive by default please forgive me, some personal drama today

https://github.com/rusefi/rusefi/issues/3989

Reply:
how do I fix this?

Reply:
change back to         case GPIOB_8: return "AUX J2 PB8";

Reply:
considering the value of this fix here and identifying https://github.com/rusefi/rusefi/issues/3989 we can just ignore the imperfection it and merge as is